Modules/Numerics/Statistics/test/itkScalarImageToTextureFeaturesFilterTest.cxx

```diff
@@ -92,7 +92,7 @@ itkScalarImageToTextureFeaturesFilterTest(int, char *[])
   for (int i = 0; i < 5; ++i)
     for (int j = 0; j < 5; j++, ++maskIt)
     {
-      maskIt.Set(bool{ 1 });
```

Comment:
Approved of course, but what about `itk::NumericTraits<bool>::OneValue()`?  😺 

https://github.com/InsightSoftwareConsortium/ITK/blob/214b763c38bc3e58d3dc96d21b0a06c13ac93863/Modules/Filtering/MathematicalMorphology/test/itkFlatStructuringElementTest2.cxx#L107

Reply:
@hjmjohnson I just "unresolved" because I did not see a "resolution" to my remark on `itk::NumericTraits<bool>::OneValue()`. What do you think?

Reply:
That's quite a round-about way to write `true`. There is no need to overcomplicate things. `itk::NumericTraits<bool>::OneValue()` exists primarily to ease implementing templated code.